build/charts/antrea/crds/traceflow.yaml

```diff
@@ -193,6 +193,8 @@ spec:
                               type: string
                             egress:
                               type: string
+                            egressNode:
+                              type: string
```

Comment:
I don't think we need to add this field for v1alpha1 version.  We have promoted the version to v1beta1 half a year ago. @tnqn @antoninbas any thoughts.

Reply:
I didn't comment because it's harmless anyway. It's just about whether v1alpha1 client will see the added field, no compatibility impact.

Reply:
removed for v1alpha1.